lib/Kconfig.debug

```diff
@@ -2224,6 +2224,9 @@ config TEST_KSTRTOX
 config TEST_PRINTF
 	tristate "Test printf() family of functions at runtime"
 
+config TEST_LONGEST_SYMBOL_SYMBOL
```

Comment:
`TEST_LONGEST_SYMBOL_SYMBOL` -> `TEST_LONGEST_SYMBOL`

Reply:
Indeed. Thanks for pointing it out @aliceinwire  . I will fix it.